### PR TITLE
Fix deprecation warnings on Swift 5.0 for Linux

### PR DESCRIPTION
**Pull request by norio-nomura**
> warning: 'launchPath' is deprecated: renamed to 'executableURL'
> warning: 'launch()' is deprecated: renamed to 'run'

### CODE REVIEW

File: Sources/Commandant/Command.swift [hunk 1]
```diff
@@ -205,10 +205,29 @@ extension CommandRegistry {
...
+				task.executableURL = URL(fileURLWithPath: path)
+				try task.run()
+			#else
+				task.launchPath = path
+				task.launch()
+			#endif
+			} catch let nserror as NSError {
+				return Int32(truncatingIfNeeded: nserror.code)
+			} catch {
+				return -1
```
Comment (mdiep) [inline Sources/Commandant/Command.swift]: Should these return `task.terminationStatus`?

Comment (norio-nomura) [reply Sources/Commandant/Command.swift]: I believe that this code block will never be executed.🤔
Should I replace it with `fatalError()` or something?

Comment (mdiep) [reply Sources/Commandant/Command.swift]: I think both `catch`s could be combined and return `task.terminationStatus`?

Comment (norio-nomura) [reply Sources/Commandant/Command.swift]: `task.run()` may throws `NSError` without executing.
https://github.com/apple/swift-corelibs-foundation/blob/master/Foundation/Process.swift#L470-L484
On such conditions, `task.terminationStatus` is `0`, so we need to catch `NSError`.

Comment (norio-nomura) [reply Sources/Commandant/Command.swift]: > On such conditions, `task.terminationStatus` is `0`

On such conditions, accessing `task.terminationStatus` produces `NSInvalidArgumentException`
```terminal.sh-session
$ swift
swift_oss_helper command enabled.
Welcome to Apple Swift version 4.2.1 (swiftlang-1000.11.42 clang-1000.11.45.1). Type :help for assistance.
  1> import Foundation
  2> let task = Process()
task: Process = {
  ObjectiveC.NSObject = {
    baseNSObject@0 = {
      isa = NSConcreteTask
    }
  }
}
  3> task.executableURL = URL(fileURLWithPath: "/dev/null")
  4> do { 
  5.     try task.run() 
  6. } catch { 
  7.     print(task.terminationStatus) 
  8. } 
2019-03-12 09:10:33.932363+0900 repl_swift[3638:58597] *** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '*** -[NSConcreteTask terminationStatus]: task not launched'
*** First throw call stack:
(
	0   CoreFoundation                      0x00007fff3a23fd49 __exceptionPreprocess + 256
	1   libobjc.A.dylib                     0x00007fff64dbda17 objc_exception_throw + 48
	2   CoreFoundation                      0x00007fff3a23fb7b +[NSException raise:format:] + 201
	3   Foundation                          0x00007fff3c427ca3 -[NSConcreteTask terminationStatus] + 104
	4   ???                                 0x00000001000fc28a 0x0 + 4296000138
	5   repl_swift                          0x0000000100000ea0 main + 0
	6   libdyld.dylib                       0x00007fff665ea3d5 start + 1
)
```

Comment (mdiep) [reply Sources/Commandant/Command.swift]: Thanks for checking! The documentation for `Process` is not good.

**Pull request merged**